hpms/src/main/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceImpl.java

```diff
@@ -76,6 +86,15 @@ private void refreshToken(long currentTimestamp) {
         authToken = authResponse.getAccessToken();
     }
 
+    private String extractCookies(MultiValueMap<String, ResponseCookie> entries) {
+        return entries.entrySet().stream()
+                .map(cookie -> cookie.getValue()
+                        .stream()
+                        .map(v -> cookie.getKey() + "=" + v.getValue()))
```

Comment:
Likewise, I have no idea what a v is.  Please give the lambda variable a more meaningful name.